tests/python/shared/fixtures/init.py

```diff
@@ -185,12 +185,30 @@ def delete_compose_files():
         filename.unlink(missing_ok=True)
 
 
+def wait_for_services():
```

Comment:
Why do we need the function?

Reply:
We are waiting for services now, not just for the server. `wait_for_server` reflects that it is an implementation detail that everything required is available via the server. I think, the latter might be not very interesting now, as we have no other checks.